Eventy/src/app/layout/nav-bar/nav-bar.component.html

```diff
@@ -7,8 +7,9 @@
   </div>
   <div>
     <!-- Router Links will be added here -->
-    <mat-icon class="tool-bar-icon clickable" [routerLink]="['profile']">account_circle</mat-icon>
-    <mat-icon class="tool-bar-icon clickable">chat_bubble</mat-icon>
-    <mat-icon class="tool-bar-icon clickable">notifications</mat-icon>
+    <mat-icon *ngIf="role !== null" class="tool-bar-icon clickable" [routerLink]="['profile']">account_circle</mat-icon>
```

Comment:
Just a quick note, I think admin should not see profile, messages or notifications?

Reply:
Actually, maybe he should see notifications but I think he won't see messages. Anyway, we should check that later!

Reply:
This is up for debate, I am not sure either! Lets discuss this together and change it later if we decide so, okay?